tests/models/test_horovod.py

```diff
@@ -71,6 +71,8 @@ def _run_horovod(trainer_options):
     ]
     if trainer_options.get("accelerator", "cpu") == "gpu":
         cmdline += ["--on-gpu"]
+    if devices == 2:
+        cmdline += ["--check-size"]
```

Comment:
I couldn't find this option in the horovodrun help page. Is this a new option?

Reply:
This is an argument for the script tests/models/data/horovod/train_default_model.py that I add in this pr. This script has an assert checking if the replication is two in here: https://github.com/PyTorchLightning/pytorch-lightning/blob/c5ff4f903f8be20173a112fc89674d7bde3c5d98/tests/models/data/horovod/train_default_model.py#L75-L76

Since some tests will have hvd.size()==1 it won't pass the test in those cases.

Reply:
ah right 🤦